src/test/java/org/apache/maven/shared/utils/cli/CommandLineUtilsTest.java

```diff
@@ -168,4 +169,25 @@ private void assertCmdLineArgs( final String[] expected, final String cmdLine )
         assertEquals( expected.length, actual.length );
         assertEquals( Arrays.asList( expected ), Arrays.asList( actual ) );
     }
+
+    @Test
+    public void testChineseEncodingIssue()
+        throws Exception
+    {
+        Commandline commandline = new Commandline( "ping www.baidu.com" );
```

Comment:
No, we cannot have tests which require outbound access. Espcially ICMP. Please change test.